pkg/caretta/links_tracer_test.go

```diff
@@ -290,15 +359,13 @@ func TestAggregations(t *testing.T) {
 	for _, test := range aggregationTests {
 		t.Run(test.description, func(t *testing.T) {
 			assert := assert.New(t)
-			m, err := createMap()
-			assert.NoError(err)
-			defer m.Close()
+			m := NewMockConnectionsMap()
 
 			tracer := caretta.NewTracerWithObjs(&MockResolver{}, m, nil)
 			pastLinks := make(map[caretta.NetworkLink]uint64)
 			var currentLinks map[caretta.NetworkLink]uint64
 			for _, connection := range test.connections {
-				m.Update(connection.connId, connection.throughput, ebpf.UpdateAny)
+				m.InnerMap[connection.connId] = connection.throughput
```

Comment:
Expose an Update() function instead - not part of the interface, but still useful instead of accessing the member directly

Reply:
added and changed to innerMap